client/store.js

```diff
@@ -12,6 +12,7 @@ export class Store {
   @observable defaultSize;
   @observable selectedSize;
   @observable showConcatenatedModulesContent = (localStorage.getItem('showConcatenatedModulesContent') === true);
+  @observable darkMode = (localStorage.getItem('darkMode') === true);
```

Comment:
The browser can disable access to `localStorage` in some environments so we should protect against `getItem` or `setItem` throwing an error. We can just fail silently and assume that dark mode is unset and thus use the system value as the default.